core/events_bubble_open.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Events fired as a result of bubble open.
+ * @author kozbial@google.com (Monica Kozbial)
+ */
+'use strict';
+
+goog.provide('Blockly.Events.BubbleOpen');
```

Comment:
We could consider moving all these events into their own folder for better organization.

Reply:
Yes, I was planning on moving the files after I get most PRs in.